airflow-core/src/airflow/api_fastapi/common/parameters.py

```diff
@@ -565,9 +583,17 @@ def float_range_filter_factory(
     def depends_float(
         lower_bound: float | None = Query(alias=f"{filter_name}_gte", default=None),
         upper_bound: float | None = Query(alias=f"{filter_name}_lte", default=None),
+        lower_bound_excluded: float | None = Query(alias=f"{filter_name}_gt", default=None),
+        upper_bound_excluded: float | None = Query(alias=f"{filter_name}_lt", default=None),
```

Comment:
Maybe we should take the opportunity to rename variables (private python interface) because it starts getting confusing in the naming IMOH.

```suggestion
        lower_bound_gte: float | None = Query(alias=f"{filter_name}_gte", default=None),
        lower_bound_gt: float | None = Query(alias=f"{filter_name}_gt", default=None),
        upper_bound_lte: float | None = Query(alias=f"{filter_name}_lte", default=None),
        upper_bound_lt: float | None = Query(alias=f"{filter_name}_lt", default=None),
```